app/src/main/java/org/simple/clinic/summary/bloodsugar/view/BloodSugarSummaryView.kt

```diff
@@ -110,10 +118,12 @@ class BloodSugarSummaryView(
     if (isInEditMode) {
       return
     }
-
     context.injector<BloodSugarSummaryViewInjector>().inject(this)
 
     delegate.prepare()
+
+    val screenDestroys: Observable<ScreenDestroyed> = detaches().map { ScreenDestroyed() }
```

Comment:
Not necessary to change this, but `ScreenDestroyed` is only used as a signal to `ReplayUntilScreenIsDestroyed` to end the stream. Since this screen is not using Mobius, we can just use `detaches()` instead.